vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -1193,6 +1193,10 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
             }
           })
         );
+
+        grid.scrollToIndex = tryCatchWrapper(function (index) {
```

Comment:
Would it work if the `queueMicrotask` was added to the invocation (in Grid and TreeGrid)? Overriding a grid function and making it async somehow feels dangerous.

Reply:
Sure, that makes sense.

Reply:
Done.